batch2/batch/batch.py

```diff
@@ -71,18 +70,8 @@
 log.info(f'INSTANCE_ID = {INSTANCE_ID}')
 log.info(f'BATCH_IMAGE = {BATCH_IMAGE}')
 
-if 'BATCH_USE_KUBE_CONFIG' in os.environ:
-    kube.config.load_kube_config()
-else:
-    kube.config.load_incluster_config()
-v1 = kube.client.CoreV1Api()
-
-app = web.Application(client_max_size=None)
-setup_aiohttp_session(app)
-
 routes = web.RouteTableDef()
 
-db = get_db()
```

Comment:
I think I also used this in instance_pool.py and driver.py

Reply:
Whatever is easiest.

Reply:
Yeah, I'll save it for later.  Rebased and unstacked, this should be ready to go now.